101-azure-bastion/azuredeploy.json

```diff
@@ -116,20 +116,7 @@
                         }
                     },
                     {
-                        "name": "bastion-subnet-out-allow",
```

Comment:
The bastion-subnet-out-allow NSG rule was intended to permit unrestricted communication between VMSS bastion instances on the AzureBastionSubnet.  Otherwise, this communication will get blocked by the VNET deny rule.  Do the VMSS bastion instances not need to communicate between themselves?

Reply:
This NSG is applied on the subnet and not on VM nic.  I doubt if the NSG applied on the subnet would block intra-vnet traffic within that subnet.

In either case, as of today, the Bastion instances do not talk to each other.